Index: b/src/Unix/linux/sigsegv_linux_arm.cpp
===================================================================
--- a/src/Unix/linux/sigsegv_linux_arm.cpp
+++ b/src/Unix/linux/sigsegv_linux_arm.cpp
@@ -209,7 +209,7 @@ buserr:
 } 
 
 static void segfault_vec(int /*sig*/, siginfo_t *sip, void *uc) {
-	mcontext_t *context = &(((struct ucontext *)uc)->uc_mcontext);
+	mcontext_t *context = &(((ucontext_t *)uc)->uc_mcontext);
 	unsigned long *regs = &context->arm_r0;
 	uintptr addr = (uintptr)sip->si_addr;
         addr -= FMEMORY;
